src/core_plugins/kbn_doc_views/kibana.json

```diff
@@ -1,4 +1,4 @@
 {
-  "name": "spy_modes",
+  "id": "kbn_doc_views",
```

Comment:
Wait, did you rename spy_modes to kbn_doc_views, or is git just mishandling this diff?

Reply:
It's just git mismatching the source files with the destination files since they are all so similar. You'll notice this file is `kbn_doc_views/kibana.json`, and git thinks it came from `spy_modes/package.json`, and below thinks that `spy_modes/kibana.json` is a brand new file... 🤷‍♀️